src/Components/UI/genreFilter/GenreFilter.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const GenreFilter = () => {
+  return (
+    <div className="genre">
+      <div className="genre__item active">all</div>
+      <div className="genre__item">documentary</div>
+      <div className="genre__item">comedy</div>
+      <div className="genre__item">horror</div>
+      <div className="genre__item">crime</div>
```

Comment:
instead of copy-pasting, please consider the following:
```
//this may be located in some separate constants file
const GENRES = ['documentary, 'comedy', 'etc'];

//...
<div>
{GENRES.map(genre => <div key={genre} className="genre__item">{genre}</div>)}
</div>
```